packages/components/psammead-topic-tags/src/__snapshots__/index.test.jsx.snap

```diff
@@ -244,8 +266,22 @@ exports[`TopicTags should correctly render a single topic tag for persian 1`] =
 }
 
 .emotion-2 a>span {
+  display: -webkit-inline-box;
+  display: -webkit-inline-flex;
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  -webkit-justify-content: center;
```

Comment:
Aye I think IE 11 support is notoriously poor for some flex properties: https://caniuse.com/?search=justify-content

Otherwise should be OK. I think we already accept quite a few visual discrepancies in IE11.